Cumulus 667 setup access logging docs (#861)
nasa/cumulus: docs/operator-docs/locating-access-logs.md

---
id: locating-access-logs
title: Locating S3 Access Logs
hide_title: true
---

# Locating S3 Access Logs

> When [enabling S3 Access Logs](../deployment/server_access_logging) for EMS Reporting you configured a `TargetBucket` and `TargetPrefix`.  Those together, are where you will find the raw S3 access logs.

Reviewer: ```suggestion
When [enabling S3 Access Logs](../deployment/server_access_logging) for EMS Reporting you configured a `TargetBucket` and `TargetPrefix`.  Inside the `TargetBucket` at the `TargetPrefix` is where you will find the raw S3 access logs.
```
